[ProPTIT] Adventure Tale/core/src/com/mygdx/game/Hud.java

```diff
@@ -112,8 +90,27 @@ public void update(){
 
     public void focusTo(int index){
         stage.getActors().get(index).setColor(0,0,0,1);
-        stage.getActors().get((index + 1) % 3).setColor(0,0,0,0.3f);
-        stage.getActors().get((index + 2) % 3).setColor(0,0,0,0.3f);
+        stage.getActors().get(index).setSize(bigButtonSize,bigButtonSize);
+        stage.getActors().get((index + 1) % 3).setColor(0,0,0,0.2f);
```

Comment:
chỗ này các em xem có code động hơn được k chứ
giờ thêm 1 nhân vật là phải thêm 2 dòng
        stage.getActors().get((index + 3) % 3).setColor(0,0,0,0.2f);
        stage.getActors().get((index + 3) % 3).setSize(buttonSize,buttonSize);

rồi lại % 4 à ? :v
anh nghĩ cái số 3 kia phải là 1 cái động tự nhận số lượng character
và mình sẽ for qua để set mấy cái này chứ nhỉ